cva6/sim/cva6.py

```diff
@@ -861,7 +857,19 @@ def load_config(args, cwd):
     if not args.testlist:
       args.testlist = cwd + "/target/"+ args.target +"/testlist.yaml"
     args.core_setting_dir = cwd + "/target/"+ args.target
-    if args.target == "rv32imc":
+    if args.target == "cv64a6_imafdc_sv39":
+      args.mabi = "lp64"
+      args.isa  = "rv64gc"
+    elif args.target == "cv32a6_imac_sv0":
+      args.mabi = "ilp32"
+      args.isa  = "rv32imac"
+    elif args.target == "cv32a6_imac_sv32":
+      args.mabi = "ilp32"
+      args.isa  = "rv32imac"
+    elif args.target == "cv32a6_imafc_sv32":
+      args.mabi = "ilp32"
+      args.isa  = "rv32imacf"
```

Comment:
Please use canonical order for `args.isa` (`rv32imafc`) to be sure compilers will recognize correctly the extensions.

Reply:
Corrected. Thanks